### PR TITLE
[libevent-2.1.10] arcticdb v4.4.0

### PR DESCRIPTION
**Pull request by IvoDD**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by IvoDD** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml
Comment (jjerphan) [inline recipe/meta.yaml:1]: The pin on fmt 10 must be removed.

**Review by jjerphan — CHANGES_REQUESTED** [review]

**Comment by IvoDD** [comment]
@conda-forge-admin, please rerender

**Comment by jjerphan** [comment]
@conda-forge-admin, please rerender

**Comment by jjerphan** [comment]
That's the first time I see an message like https://github.com/conda-forge/arcticdb-feedstock/pull/192#issuecomment-2036623715.

It might be explained by your absence in the recipe maintainers on the `libevent-2.1.10` branch which is corrected by a7ac933.

**Review by jjerphan — APPROVED** [review]
LGTM given :green_circle: CI checks.

**Review by alexowens90 — APPROVED** [review]

**Pull request merged**